data/community/members.yaml

```diff
@@ -0,0 +1,2211 @@
+technical-committee:
+  - name: jsuereth
+    html_url: https://github.com/jsuereth
+    avatar_url: https://avatars.githubusercontent.com/u/29006?v=4
+  - name: carlosalberto
+    html_url: https://github.com/carlosalberto
+    avatar_url: https://avatars.githubusercontent.com/u/260065?v=4
+  - name: bogdandrutu
+    html_url: https://github.com/bogdandrutu
+    avatar_url: https://avatars.githubusercontent.com/u/1373887?v=4
+  - name: lmolkova
+    html_url: https://github.com/lmolkova
+    avatar_url: https://avatars.githubusercontent.com/u/2347409?v=4
+  - name: yurishkuro
+    html_url: https://github.com/yurishkuro
+    avatar_url: https://avatars.githubusercontent.com/u/3523016?v=4
+  - name: jmacd
+    html_url: https://github.com/jmacd
+    avatar_url: https://avatars.githubusercontent.com/u/3629705?v=4
+  - name: tigrannajaryan
+    html_url: https://github.com/tigrannajaryan
+    avatar_url: https://avatars.githubusercontent.com/u/4194920?v=4
+  - name: arminru
+    html_url: https://github.com/arminru
+    avatar_url: https://avatars.githubusercontent.com/u/7052238?v=4
+  - name: reyang
+    html_url: https://github.com/reyang
+    avatar_url: https://avatars.githubusercontent.com/u/17327289?v=4
+  - name: jack-berg
+    html_url: https://github.com/jack-berg
+    avatar_url: https://avatars.githubusercontent.com/u/34418638?v=4
+spec-sponsors:
+  - name: tedsuo
+    html_url: https://github.com/tedsuo
+    avatar_url: https://avatars.githubusercontent.com/u/24074?v=4
+  - name: jsuereth
+    html_url: https://github.com/jsuereth
+    avatar_url: https://avatars.githubusercontent.com/u/29006?v=4
+  - name: carlosalberto
+    html_url: https://github.com/carlosalberto
+    avatar_url: https://avatars.githubusercontent.com/u/260065?v=4
+  - name: Oberon00
+    html_url: https://github.com/Oberon00
+    avatar_url: https://avatars.githubusercontent.com/u/849039?v=4
+  - name: bogdandrutu
+    html_url: https://github.com/bogdandrutu
+    avatar_url: https://avatars.githubusercontent.com/u/1373887?v=4
+  - name: dyladan
+    html_url: https://github.com/dyladan
+    avatar_url: https://avatars.githubusercontent.com/u/1612643?v=4
+  - name: lmolkova
```

Comment:
Looks like TC list is duplicated into spec-sponsors. Is that the intend? I think its best to remove TC from spec-sponsor list.

Reply:
good catch! Let me fix that